server/controllers/driverControllers.js

```diff
@@ -72,28 +72,36 @@ const getRoutes = async (req, res) => {
     }
   });
 
+  routes = routes.filter(route => {
+    const routeDate = new Date(`${route.dataValues.date} ${route.dataValues.departure}`);
+
+    return routeDate >= currentDate;
+  });
+
+  clearOldRoutes();
+
+  res.send(routes);
+};
+
+const clearOldRoutes = async (req, res) => {
+  const routes = await Route.findAll();
+  const currentDate = new Date();
+
```

Comment:
I think we could filter old route id in filter too. So we don't have to repeat findAll and iterate routes.